static/docs/user-guide/install/dev.md

```diff
@@ -19,7 +14,9 @@ $ pip install git+https://github.com/iterative/dvc.git
 > (e.g. `0.24.3-9c7381`). For more information on naming convention, refer to
 > [dvc version](/doc/command-reference/version).
 
-## Install branch from GitHub
+Note that this will automatically upgrade your system's DVC version to the
```

Comment:
this is not true in many many case when DVC was not installed via pip.

Reply:
@dashohoxha are you going to address this? I'm not merging other PRs until we merge this one to avoid the conflicts, but we can't wait too long. Let me know if you need our help with this.

Reply:
> this is not true in many many case when DVC was not installed via pip.

@shcheklein It is a verbatim copy of this: https://dvc.org/doc/user-guide/development

> Let me know if you need our help with this.

I'd prefer if someone else rewords it, because I am afraid that no matter what I do, it will never be good enough.

> I'm not merging other PRs until we merge this one to avoid the conflicts

In this case there are not likely to be any conflicts, since this PR only adds new pages, without touching any existing ones.

Reply:
@dashohoxha 

> @shcheklein It is a verbatim copy of this: https://dvc.org/doc/user-guide/development

kk. I think that document is just outdated a bit (or wrong that happens a lot).

> I'd prefer if someone else rewords it

kk

>

putting this PR on hold until I have time to address the comments that left. Merging everything other PRs that are ready.

Reply:
> this is not true in many many case when DVC was not installed via pip

I have tried to fix it. I am not sure this is best way to fix it, but maybe it is acceptable.